examples/experimental/dagster-airlift/README.md

```diff
@@ -145,79 +169,102 @@ defs = Definitions.merge(
 To build assets for our dbt invocation, we can use the Dagster-supplied factory `dbt_defs`, installable via `uv pip install dagster-airlift[dbt]`. This will load each dbt model as its own asset:
 
+`airlift.py`
 ```python
-from dagster import AssetSpec
-from dagster_airlift.core import build_defs_from_airflow_instance, dag_defs, task_defs
+import os
+from pathlib import Path
+
+from dagster import AssetSpec, Definitions
+from dagster_airlift.core import (
+    AirflowInstance,
+    BasicAuthBackend,
+    build_defs_from_airflow_instance,
+    dag_defs,
+    task_defs,
+)
+from dagster_airlift.dbt import dbt_defs
 from dagster_dbt import DbtProject
 
-defs = build_defs_from_airflow_instance(
-    airflow_instance=airflow_instance,
-    defs=dag_defs(
+
+def dbt_project_path() -> Path:
+    env_val = os.getenv("DBT_PROJECT_DIR")
+    assert env_val
+    return Path(env_val)
+
+
+def rebuild_customer_list_defs() -> Definitions:
+    return dag_defs(
         "rebuild_customers_list",
         task_defs(
             "load_raw_customers",
-            AssetSpec(key=["raw_data", "raw_customers"])
-        ),
-        # encode dependency on customers output
-        task_defs(
-            "export_customers",
-            AssetSpec(key="customers_csv", deps=["customers"])
+            Definitions(
+                assets=[
+                    AssetSpec(key=["raw_data", "raw_customers"]),
+                ]
+            ),
         ),
         task_defs(
             "build_dbt_models",
+            # load rich set of assets from dbt project
             dbt_defs(
                 manifest=dbt_project_path() / "target" / "manifest.json",
                 project=DbtProject(dbt_project_path()),
             ),
         ),
+        task_defs(
+            "export_customers",
+            Definitions(
+                assets=[
+                    # encode dependency on customers output
+                    AssetSpec(key="customers_csv", deps=["customers"]),
+                ]
+            ),
+        ),
+    )
+
+
+defs = build_defs_from_airflow_instance(
+    airflow_instance=AirflowInstance(
+        auth_backend=BasicAuthBackend(
+            webserver_url="http://localhost:8080",
+            username="admin",
+            password="admin",
+        ),
+        name="airflow_instance_one",
     ),
+    defs=rebuild_customer_list_defs(),
 )
 ```
 
-### Mapping assets to tasks
-
-The `from_task` and `from_dag` utilities link created assets to their underlying tasks. Assets which are properly linked will be materialized by the Airlift sensor once the corresponding task completes.
 
 ### Viewing observed assets
 
 Once your assets are set up, you should be able to reload your Dagster definitions and see a full representation of the dbt project and other data assets in your code.
 
-Kicking off a run of the DAG, you should see the newly created assets materialize in Dagster.
+<p align="center">
+
+![Observed asset graph in Dagster](./images/observe.svg)
```

Comment:
how are you making these svgs these are great

Reply:
Bottom button in the asset graph! 
![Screenshot 2024-08-27 at 1 20 24 PM](https://github.com/user-attachments/assets/55558c22-bb7b-4e62-a33f-301e7b0eb301)
great feature

Reply:
TIL!